infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/receive/LogReplicationMetadataManager.java

```diff
@@ -508,6 +508,13 @@ public void updateSyncStatus(String clusterId, SyncType lastSyncType, SyncStatus
 
             CorfuStoreEntry<ReplicationStatusKey, ReplicationStatusVal, Message> record = txn.getRecord(replicationStatusTable, key);
 
+            // the condition is true when FSM to a remote cluster has been shutdown.
```

Comment:
nit - would it be better to make this comment a bit elaborate?  Something like -
'... when a remote cluster is removed from the topology and FSMs to it have been shutdown'